graphrag/config/models/graph_rag_config.py

```diff
@@ -224,20 +224,20 @@ def _validate_update_index_output_base_dir(self) -> None:
     )
     """The basic search configuration."""
 
-    vector_store: VectorStoreConfig = Field(
-        description="The vector store configuration.", default=VectorStoreConfig()
+    vector_store: dict[str, VectorStoreConfig] = Field(
+        description="The vector store configuration.",
+        default={"default": VectorStoreConfig()},
```

Comment:
It looks like you used "output" as the default in defaults.py, so I would use that variable here. I'd probably make sure the default name "default" isn't used anywhere since it could conflict with keywords (I'm thinking maybe JSON versions of settings because `default` is a keyword in JavaScript?)